pkg/dashboards/health.go

```diff
@@ -1,77 +1,108 @@
 package dashboards
 
 import (
-	_ "embed"
-	"encoding/json"
-	"strings"
+	"fmt"
 )
 
-//go:embed infra_health_dashboard.json
-var healthDashboardEmbed string
+func CreateHealthDashboard(netobsNs string, _ []string) (string, error) {
+	d := Dashboard{Title: "NetObserv / Health"}
 
-func CreateHealthDashboard(netobsNs string, metrics []string) (string, error) {
-	var result map[string]any
-	err := json.Unmarshal([]byte(healthDashboardEmbed), &result)
-	if err != nil {
-		return "", err
-	}
+	// Global stats
+	d.Rows = append(d.Rows, NewRow("", false, "100px", []Panel{
+		NewSingleStatPanel("Flows per second", PanelUnitShort, 3, NewTarget(
+			"sum(rate(netobserv_agent_evicted_flows_total[1m]))", "")),
+		NewSingleStatPanel("Sampling", PanelUnitShort, 3, NewTarget(
+			"avg(netobserv_agent_sampling_rate)", "")),
+		NewSingleStatPanel("Errors last minute", PanelUnitShort, 3, NewTarget(
+			`sum(increase(netobserv_agent_errors_total[1m]))
+			+ sum(increase(controller_runtime_reconcile_errors_total{job="netobserv-metrics-service"}[1m]))`, "")), // TODO: add FLP errors here
+		NewSingleStatPanel("Loki drops per second", PanelUnitShort, 3, NewTarget(
+			"sum(rate(netobserv_loki_dropped_entries_total[1m]))", "")),
+	}))
 
-	// return dashboard as is if not containing rows
-	if result["rows"] == nil {
-		return healthDashboardEmbed, nil
-	}
+	// FLP stats
+	// TODO: FLP error
+	flpStats := NewRow("Flowlogs-pipeline statistics", false, "250px", []Panel{
+		NewGraphPanel("Flows per second", PanelUnitShort, 6, false, []Target{
+			NewTarget("sum(rate(netobserv_ingest_flows_processed[1m]))", "Flows ingested"),
+			NewTarget("sum(rate(netobserv_loki_sent_entries_total[1m]))", "Flows sent to Loki"),
+			NewTarget("sum(rate(netobserv_loki_dropped_entries_total[1m]))", "Flows dropped due to Loki error"),
+		}),
+	})
+	//	if slices.Contains(metrics, "namespace_flows_total") {
```

Comment:
note, I left comments in code as I'm not sure yet what's the best option: to display charts with unavailable metrics or not? Currently we don't display them, and here I'm suggesting to still display them. The downside if we hide empty charts is that it breaks the design of having 2 charts per row. Also we could argue that showing empty charts helps the user know that they could have it with an appropriate configuration